colossalai/utils/profiler/common.py

```diff
@@ -0,0 +1,75 @@
+from enum import Enum
+from .comm_profiler import CommProfiler
+from typing import List
+from torch.utils.tensorboard import SummaryWriter
+
+
+class ProfilerEvent(Enum):
+    '''
+    Event types to profile
+    '''
+    COMM = CommProfiler

```

Comment:
why is profiler an event?

Reply:
It is to distinguish events (or activities? as named by torch) to profile (e.g. communication, memory), and each event relates to a profiler

Reply:
It is kind of confusing as in communication profiler, it has defined its own event.

Reply:
So change it to 'ProfilerActivity'?

Reply:
ok